app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_density">Display Density:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_window_size">Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_mouse">Mouse</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_touch">Touch</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_edit">Edit</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_save">Save</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_dpi">Display DPI:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_max_window_size">Max. Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa">MSAA</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_disabled">Disabled</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_2">2x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_4">4x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_desktop">Desktop</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_mobile">Mobile</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_vr">VR</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset">Reset Developer Settings</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset_button">Reset</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_environments">Environment</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_meadow">Meadow</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_cave">Cave</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_void">Void</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_color">Pointer Color</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_white">White</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_purple">Purple</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="on">ON</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="off">OFF</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_start_1">What would you like to</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_start_2">search on the web?</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_example">Example: Please say, \"360 videos, weather, news…\"</string>
```

Comment:
why does this string have to contain unescaped quotation marks? can we do the (un)escaping in the Java code?

Reply:
@keianhzo Do you know why?